### PR TITLE
Technical documentation: add Prometheus docs

### PR DESCRIPTION
**Pull request by Stadicus**

### CODE REVIEW

File: docs/support/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+There's a lot going on within the BitBox Base and it's good to have reliable metrics over time to aid in development, fine-tuning, during operations and when looking for the cause of a specific error.
+[Prometheus](https://prometheus.io/) takes care of that.
+It is a open-source monitoring solution that is easily extendable to monitor any application.
+Measurements are stored in a time-series database that can easily be queried and visualized, e.g. using [Grafana](grafana.md).
+This database is also the main data pool that is queried by the Base Middleware.
+
+### Installation
+
+The Prometheus archive file is downloaded directly from the GitHub releases page and checked against a hardcoded checksum.
```
Comment (hkjn) [inline docs/support/prometheus.md]: Maybe it's just me, but "archive file" sounds more like it's about the data used to configure Prometheus, whereas we are describing the application itself here. Perhaps just "Prometheus is downloaded directly from the GitHub releases page [...]"?

File: docs/support/prometheus.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  - job_name: lightningd
+    static_configs:
+    - targets: ['127.0.0.1:9900']
+```
+
+### Metrics
+
+The following metrics are collected both from the system and from specific applications.
+
+* **Operating System**: the [Prometheus Node Exporter](https://github.com/prometheus/node_exporter) collects very granualar system operations information.
```
Comment (hkjn) [inline docs/support/prometheus.md]: Nit: s/`granualar`/`granular`.

Comment (hkjn) [inline docs/support/prometheus.md]: Nit: Maybe it'd be more readable to say "collects very granular information about the system, such as CPU and memory usage".

File: docs/support/prometheus.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+layout: default
+title: Prometheus
+parent: Supporting Applications
+nav_order: 750
+---
+## Prometheus: System Monitoring
+
+There's a lot going on within the BitBox Base and it's good to have reliable metrics over time to aid in development, fine-tuning, during operations and when looking for the cause of a specific error.
+[Prometheus](https://prometheus.io/) takes care of that.
+It is a open-source monitoring solution that is easily extendable to monitor any application.
```
[the remark this answers is not in the document]
Comment (Stadicus) [reply docs/support/prometheus.md]: ACK

File: docs/support/prometheus.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+Measurements are stored in a time-series database that can easily be queried and visualized, e.g. using [Grafana](grafana.md).
+This database is also the main data pool that is queried by the Base Middleware.
```
[the remark this answers is not in the document]
Comment (Stadicus) [reply docs/support/prometheus.md]: ACK

File: docs/support/prometheus.md (hunk 1 above)
Comment (Stadicus) [reply docs/support/prometheus.md]: ACK

File: docs/support/prometheus.md (hunk 2 above)
Comment (Stadicus) [reply docs/support/prometheus.md]: ACK

Comment (Stadicus) [reply docs/support/prometheus.md]: ACK

**Comment by Stadicus** [comment]
ACK for all suggested changes

File: docs/support/prometheus.md [hunk 5]
```diff
@@ -8,14 +8,14 @@ nav_order: 750
...
+It is a open-source monitoring solution that is easily extensible to monitor any application.
 Measurements are stored in a time-series database that can easily be queried and visualized, e.g. using [Grafana](grafana.md).
-This database is also the main data pool that is queried by the Base Middleware.
+This database is also the main source of data that is queried by the Base Middleware.
 
 ### Installation
 
-The Prometheus archive file is downloaded directly from the GitHub releases page and checked against a hardcoded checksum.
-The archive is extracted, its content copied to various system locations and proper ownerwhip is set while building the Armbian image.
+The Prometheus is downloaded directly from the GitHub releases page and checked against a hardcoded checksum.
```
Comment (hkjn) [inline docs/support/prometheus.md]: Nit: "The Prometheus" sounds slightly odd.. would it be clearer to say "Prometheus" (no "the")?

**Pull request merged**